README.md

```diff
@@ -43,6 +43,11 @@ components: { BackToTop }
 | bottom        | String           | 40px            | bottom position of the component                                         |
 | right         | String           | 30px            | right position of the component                                          |
 
+## Events
+
+| Name                   | Description                                                              |
+|------------------------|--------------------------------------------------------------------------|
+| triggered              | Fired when back to top element is clicked                                |
```

Comment:
Hey @ventralnet,
Thanks for the issue and the PR ;D

This feature is amazing!!

About the event name, what you think if we use `scrolled` and the description `Fired when page's scroll end` ?